src/init.ts

```diff
@@ -1,17 +1,17 @@
 import { DEFAULT_DOMAIN, JAAS_DOMAIN } from './constants';
-import IJitsiMeetExternalApi from './types/IJitsiMeetExternalApi';
+import { JitsiMeetExternalApi } from './types';
 
 type ExternalApi = {
     isLoaded: boolean;
-    callbacks: ((err: Error | null, api?: new () => IJitsiMeetExternalApi) => void)[];
+    callbacks: ((err: Error | null, api?: JitsiMeetExternalApi) => void)[];
```

Comment:
I would name this JitsiMeetApi or even JitsiMeetExternalApi instead of `api`. We conventionally name `api` the instance, not the class definition.